dataset.py

```diff
@@ -345,13 +345,8 @@ def test_compute_mean_std(dataset):
         compute_mean_std(dataset)
         return
 
-    transform=transforms.Compose([
-        transforms.ToPILImage(),
-        transforms.ToTensor()])
-
-    data_dir = '/home/hongshan/data/Trees_processed/'
-    ds = TreeDataset(data_dir, transform=transform, elv_transform=transform,
-            mask_transform=transform, purpose=None)
-    mean,std = compute_mean_std(ds)
-    print(mean, std)
+
+    data_dir = '/mnt/efs/Trees_processed/'
```

Comment:
Why are you using the EFS instead of S3? That said, I like EFS as you can use it from different instances, so I'm more curious then anything else :)

Reply:
Hey Matt, 
Didn't see your reply until now. Yeah I used EFS precisely because of that. I mounted my EFS to one cpu instance and one gpu instance. I do all the coding on the cpu instance. when it is good to 
train I just switch to the gpu instance